azure-pipelines.yml

```
@@ -90,7 +90,7 @@ stages:
         displayName: "Build NuGet packages"
         inputs:
           command: pack
-          packagesToPack: 'analyzers/src/**/*.nuspec'
+          packagesToPack: 'analyzers/src/**/*.nuspec;analyzers/packaging/**/*.nuspec'
```

AFAICT there are no `nuspec` files in `src` now
```suggestion
          packagesToPack: 'analyzers/packaging/**/*.nuspec'
```

---

This will skip the CFG package generation.

---

Or like this
```suggestion
          packagesToPack: 'analyzers/src/**/*.nuspec;analyzers/packaging/*.nuspec'
```